baybe/surrogates/gaussian_process.py

```diff
@@ -72,29 +72,31 @@ def _fit(self, searchspace: SearchSpace, train_x: Tensor, train_y: Tensor) -> No
             train_x.shape[-1] >= 50
         )
 
+        # TODO Until now, only the kernels use our custom priors, hence the explicit
+        # to_gpytorch() calls for all others
         # low D priors
         if train_x.shape[-1] < 10:
             lengthscale_prior = [GammaPrior(1.2, 1.1), 0.2]
-            outputscale_prior = [GammaPrior(5.0, 0.5), 8.0]
-            noise_prior = [GammaPrior(1.05, 0.5), 0.1]
+            outputscale_prior = [GammaPrior(5.0, 0.5).to_gpytorch(), 8.0]
```

Comment:
wouldnt it make sense to shift the `to_gpytorch` calls to the code below where they are actually passed to a gpytorch function. This here feels very messy due to the mixing

Reply:
I did that on purpose to make clear that we still technically require these objects to be `gpytorch` objects. But I will simply move these calls to the places where the conversion needs to happen and then resolve.